plenum/server/config_req_handler.py

```diff
@@ -83,8 +117,8 @@ def _state_path_taa_digest(digest: str):
         return "taa:d:{digest}".format(digest=digest).encode()
 
     @staticmethod
-    def _taa_digest(version: str, text: str) -> str:
-        return sha256('{}{}'.format(version, text).encode()).hexdigest()
+    def _taa_digest(version: str, text: str) -> bytes:
+        return sha256('{}{}'.format(version, text).encode()).hexdigest().encode()
```

Comment:
Do we really need conversion to bytes here? As far I as see this adds two additional conversions - here, to bytes, and in `update_txn_author_agreement` - back to string to be passed to `_state_path_taa_digest`. Alternatively - should we make `_state_path_taa_digest` accept bytes as well, instead of string?

Reply:
updated